src/ImageSharp/Common/Helpers/SimdUtils.HwIntrinsics.cs

```diff
@@ -112,9 +112,9 @@ public static void Shuffle4Reduce(
             ref Span<byte> destination,
             [ConstantExpected] byte control)
         {
-            if ((Vector512.IsHardwareAccelerated && Vector512Utilities.SupportsShuffleByte) ||
-                (Vector256.IsHardwareAccelerated && Vector256Utilities.SupportsShuffleByte) ||
-                (Vector128.IsHardwareAccelerated && Vector128Utilities.SupportsShuffleByte))
+            if ((Vector512.IsHardwareAccelerated && Vector512_.SupportsShuffleNativeByte) ||
+                (Vector256.IsHardwareAccelerated && Vector256_.SupportsShuffleByte) ||
```

Comment:
Noticed this middle one is `SupportsShuffleByte` while the other two are `SupportsShuffleNativeByte`, guessing that was a mistake?